From c92669986eb3f899905224e8bcb78460e655bf5b Mon Sep 17 00:00:00 2001
From: Ivan Vecera <ivecera@redhat.com>
Date: Wed, 31 May 2017 15:44:34 +0200
Subject: [PATCH 166/169] tc: add missing limits.h header
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1435624
Upstream status: iproute2.git commit d421bb4efea64

commit d421bb4efea6470c1667be3e8bfb1529e8e832d0
Author: Baruch Siach <baruch@tkos.co.il>
Date:   Thu Dec 22 20:52:48 2016 +0200

    tc: add missing limits.h header

    This fixes under musl build issues like:

    f_matchall.c: In function ‘matchall_parse_opt’:
    f_matchall.c:48:12: error: ‘LONG_MIN’ undeclared (first use in this function)
       if (h == LONG_MIN || h == LONG_MAX) {
                ^
    f_matchall.c:48:12: note: each undeclared identifier is reported only once for each function it appears in
    f_matchall.c:48:29: error: ‘LONG_MAX’ undeclared (first use in this function)
       if (h == LONG_MIN || h == LONG_MAX) {
                                 ^

    Signed-off-by: Baruch Siach <baruch@tkos.co.il>

Signed-off-by: Ivan Vecera <ivecera@redhat.com>
---
 tc/tc_util.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tc/tc_util.h b/tc/tc_util.h
index 4f544363..3e0da966 100644
--- a/tc/tc_util.h
+++ b/tc/tc_util.h
@@ -2,6 +2,7 @@
 #define _TC_UTIL_H_ 1
 
 #define MAX_MSG 16384
+#include <limits.h>
 #include <linux/pkt_sched.h>
 #include <linux/pkt_cls.h>
 #include <linux/gen_stats.h>
-- 
2.13.0

